about.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://fonts.cdnfonts.com/css/lato" rel="stylesheet" />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <link rel="stylesheet" href="./css/styles.css" />
+    <title>About NAMM SHOW(EAST AFRICAN)</title>
+  </head>
+  <body>
+    <header class="abt-header">
+      <nav class="mob-nav">
+        <i class="fa-sharp fa-solid fa-bars open_btn"></i>
+        <div class="mob-menu">
+          <i class="fa-sharp fa-solid fa-xmark close_btn"></i>
+          <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="index.html">Program</a></li>
+            <li><a href="about.html">Join</a></li>
+            <li><a href="about.html">Sponsor</a></li>
+            <li><a href="about.html">News</a></li>
+          </ul>
+        </div>
+      </nav>
+      <nav class="desk_nav">
+        <div class="top-nav">
+          <i class="fa-brands fa-facebook-f"></i>
+          <i class="fa-brands fa-twitter"></i>
+          <ul>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+        </div>
+        <div class="desktop-nav">
+          <a href="index.html"
+            ><img src="assets/NAMMlogo-2.jpg" alt="NAMM Logo"
+          /></a>
+          <ul>
+            <li><a href="about.html">About</a></li>
+            <li><a href="index.html">Program</a></li>
+            <li><a href="about.html">Join</a></li>
+            <li><a href="about.html">Sponsor</a></li>
+            <li><a href="about.html">News</a></li>
+            <li class="connect">
+              <a href="https://www.namm.org/contact">Get Connected</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <section class="intro-section">
+        <div class="name_title_abt">
+          <h3 class="intro-text-abt">"Hello! Musical World"</h3>
+          <h1 class="mn-title-abt">
+            <span> NAMM Show </span>
+          </h1>
```

Comment:
Kindly make reference to the comments I dropped in the intro section of the index page, as the same issue also applies here😉😉